### ISSUE TITLE
Conda install fails

### ISSUE THREAD

**Issue by Ben7124** [institut-de-genomique/HAPO-G#18]
Hello,
I get the following error when I use conda install. Can you help me solve it? Thanks!

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - hapog -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']

**Comment by bistace** [comment]
Hello,
could you please indicate the commands you used to create the conda environment and install hapog? I just checked on two separate environments and do not encounter your issue.

**Comment by Ben7124** [comment]
Hello,
I used the exact commands on the install page. Full output after install command is below. OS is Ubuntu 22.04 LTS (GNU/Linux 5.15.0-40-generic x86_64). Thank for your help!
conda create -n hapog
conda activate hapog
conda install -c lbgb_cea hapog

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - hapog -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.35
